ci-operator/config/coreos/coreos-assembler/coreos-coreos-assembler-main.yaml

```diff
@@ -1,19 +1,45 @@
-binary_build_commands: make && make check
+base_images:
+  fedora:
+    name: fedora
+    namespace: coreos
+    tag: "34"
 build_root:
   image_stream_tag:
-    name: cosa-buildroot
+    name: fcos-buildroot
     namespace: coreos
-    tag: latest
+    tag: testing-devel
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    fedora:
+      as:
+      - registry.svc.ci.openshift.org/coreos/fedora:34
```

Comment:
Do we actually need this? Would it make sense to just drop it so we let it default to the `FROM` in the Dockerfile so that we don't have to bump it each time?